tdrs-backend/tdpservice/parsers/migrations/0002_alter_parsererror_error_type.py

```diff
@@ -15,4 +15,13 @@ class Migration(migrations.Migration):
             name='error_type',
             field=models.TextField(choices=[('1', 'File pre-check'), ('2', 'Record value invalid'), ('3', 'Record value consistency'), ('4', 'Case consistency'), ('5', 'Section consistency'), ('6', 'Historical consistency')], max_length=128),
         ),
+        migrations.CreateModel(
```

Comment:
I just realized that this migration has to be moved to a new migration. We can't add to an existing migration